<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndividualChannelInfo</class>
 <widget class="QWidget" name="IndividualChannelInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Individual Channel Information</string>
  </property>
  <widget class="QTextEdit" name="InputElectrodeNumber">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Electrode No.</string>
   </property>
   <property name="buddy">
    <cstring>InputElectrodeNumber</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Row</string>
   </property>
   <property name="buddy">
    <cstring>InputElectrodeRow</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Col</string>
   </property>
   <property name="buddy">
    <cstring>InputElectrodeCol</cstring>
   </property>
  </widget>
  <widget class="QTextEdit" name="InputElectrodeRow">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="InputElectrodeCol">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelElectrodeInfo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Extra information</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>40</y>
     <width>641</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="PlotWidget" name="AmplitudeHistPlot" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="PlotWidget" name="ChannelTracePlot" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="PlotWidget" name="SpikeRateHistPlot" native="true"/>
    </item>
    <item row="1" column="1">
     <widget class="PlotWidget" name="SpikeRatePlot" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
